meta-lmp-bsp/recipes-support/mfgtool-files/mfgtool-files_%.bbappend

```diff
@@ -64,6 +64,13 @@ do_deploy_prepend_apalis-imx6() {
 }
 
 do_compile_append_apalis-imx6-sec() {
+    sed -i -e '/^FB: ucmd setenv.*/i\
+\
+# Clear fiovb vars \
+FB: ucmd fiovb init 0 \
+FB[-t 50000]: ucmd fiovb write_pvalue bootcount 0 \
+FB[-t 50000]: ucmd fiovb write_pvalue rollback 0\n' full_image.uuu
+
```

Comment:
We have a dynamic logic that can be used to identify when fiovb or uboot env will be used/required at https://github.com/foundriesio/meta-lmp/blob/master/meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common.cmd.in#L31 (via imx_is_closed), maybe we can leverage that and have full_image.uuu work on both cases?

That way we can avoid maintaining this sed here.

Reply:
done